client/state/sites/hooks/use-site-global-styles-status.ts

```diff
@@ -62,19 +46,16 @@ const getGlobalStylesInfoForSite = ( siteId: number | null ): Promise< GlobalSty
 		return Promise.resolve( {
 			shouldLimitGlobalStyles: true,
 			globalStylesInUse: false,
-			globalStylesInPersonalPlan: false,
+			globalStylesInPersonalPlan: true,
```

Comment:
This should either be false, or you can remove this property since it won't be needed.

Reply:
How come? I'm not yet removing all of the code that depends upon `globalStylesInPersonalPlan` because then I'll have to do all of that work once for control and once for treatment and then over the next four weeks those PRs will inevitably become outdated and need rebasing and testing all over again.

Reply:
if `globalStylesInPersonal` doesn't exist then the code will use `shouldLimitGlobalStyles` which should already be returning if the GS should be blocked.

Reply:
@rcrdortiz `shouldLimitGlobalStyles` is not the same as `globalStylesInPersonal`:
- `shouldLimitGlobalStyles`: controls whether Limited GS notices should appear
- `globalStylesInPersonal`:  controls the plan we're upselling in the Limited GS notices

In the long term (and if `control` wins), we'll be removing any code that uses `globalStylesInPersonal` and always upsell the Premium plan. 

Before doing that, @dsas preferred to open this PR which just hardcodes `globalStylesInPersonal` to true so we can quickly implement the experiment recommendations and leave the full clean-up task for later.

Reply:
More context about the reasoning of this initial work [here](https://github.com/Automattic/wp-calypso/pull/79682#discussion_r1269379459):

> I'm trying to minimise the work needed to initially implement the experiment recommendations so we can do that quickly and then clean-up more afterwards. Removing this function now would mean hard-coding true + a comment in a few places, it seemed cleaner to just leave it here for now.

Reply:
When/if treatment wins then `shouldLimitGlobalStyles` will have the information that determines if we should upsell or not. But if you guys prefer this approach then go ahead.

Reply:
I didn't especially want interim changes either :-/ . I just preferred that over the alternative - preparing two lots of work now, knowing that one lot will definitely get thrown away and the other lot will need rebasing and crossed fingers that any conflicts are simple.

Reply:
> preparing two lots of work now

Why do we want to prepare the work now instead of waiting for the experiment to finish and implement only what's needed? I don't think it would take too much time.